packages/asc-ui/src/components/TopBar/StyledTopBar.ts

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components'
+
+const defaultBackgroundColor = '#fff'
```

Comment:
Use colors from theme and preferably create an object called defaultProps { ... }

Reply:
done